File: CSharp/UnicodeFileToHtmlTextConverter/HttpUtility.cs
namespace TDDMicroExercises.UnicodeFileToHtmlTextConverter;

class HttpUtility
{
    public static string HtmlEncode(string line)
    {
        line = line.Replace("<", "&lt;");
        line = line.Replace(">", "&gt;");
        line = line.Replace("&", "&amp;");
        line = line.Replace("\"", "&quot;");
        line = line.Replace("\'", "&quot;");
        return line;
    }
}

File: TypeScript/text-converter/http-utility.ts

export default function htmlEncode(line: string): string {
  line = line.replace(/</g, "&lt;");
  line = line.replace(/>/g, "&gt;");
  line = line.replace(/&/g, "&amp;");
  line = line.replace(/"/g, "&quot;");
  line = line.replace(/'/g, "&quot;");
  return line;
}
